### PR TITLE
[ZH] Update help centre articles

### PR DESCRIPTION
**Pull request by Kotoki1337**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Help_centre/Account_restrictions/zh.md [hunk 1]
```diff
@@ -1,15 +1,10 @@
...
-
 # 账号限制
 
-*主页面： [帮助中心](/wiki/Help_centre)*
+*主页面：[帮助中心](/wiki/Help_centre)*
 
 账号限制是社区对表现出不正常、可疑、或违规行为的账号的一个强制措施。
 
-账号被限制后，将无法与社区进行互动，直到限制解除为止。账号被限制的玩家仍然可以继续游玩，下载谱面并上传成绩，但账号的信息不会被他人看见。
+账号被限制后，将无法与社区进行互动，直到限制解除为止。账号被限制的玩家仍然可以继续游玩，下载谱面并上传成绩，但账号的个人信息不会被他人可见。
```
Comment (kj415j45) [inline wiki/Help_centre/Account_restrictions/zh.md:7]:
```suggestion
账号被限制后，将无法与社区进行互动，直到限制解除为止。账号被限制的玩家仍然可以继续游玩，下载谱面并上传成绩，但账号的个人信息变为不可见。
```

Comment (Kotoki1337) [reply wiki/Help_centre/Account_restrictions/zh.md:7]: 读起来舒服一点

```suggestion
账号被限制后，将无法与社区进行互动，直到限制解除为止。账号被限制的玩家仍然可以继续游玩，下载谱面并上传成绩，但账号的个人信息会变为不可见。
```

File: wiki/Help_centre/Account_restrictions/zh.md [hunk 2]
```diff
@@ -106,17 +109,19 @@ outdated_since: b830432c4768e242ce32ad592010de960bbefdeb
 
 请您知晓：向这个地址发邮件只能保证对您的申诉的复审或应答。如果您被发现不符合申诉过程的期望，复审不会保证申诉结果的改变。
 
-**我们只会对发送至 [appeal-review@ppy.sh](mailto:appeal-review@ppy.sh) 的复审或调查请求进行答复。** 不要试图通过推特，Discord，QQ 等方式联系开发人员（以及 GMT、NAT 等志愿者团队）来绕过这个政策——他们帮不了您。
+**我们只会对发送至 [appeal-review@ppy.sh](mailto:appeal-review@ppy.sh) 的复审或调查请求进行答复。**不要试图通过推特，Discord，QQ 等方式联系开发人员（以及 GMT、NAT 等志愿者团队）来绕过这个政策——他们帮不了您。
```
Comment (cl8n) [inline wiki/Help_centre/Account_restrictions/zh.md:112]: should probably specifically mention Twitter as in EN as that's where some people on these volunteer teams have accounts

File: wiki/Help_centre/Account_restrictions/zh.md [hunk 3]
```diff
@@ -38,46 +33,46 @@ outdated_since: b830432c4768e242ce32ad592010de960bbefdeb
...
+- **任何严重违规行为都将立即导致您再次被限制。**包括严重的禁言。
 
 此外，我们在一年以内不会通过任何修改您的账号的国籍的申请。
 
 有些时候，一个被解除限制的账号会禁止作为选手或组织团队成员参加官方的或其它奖励徽章的竞赛。禁赛的时长为每次违规 1 年（在第二次申诉后，会禁赛 2 年）。解封邮件会写明禁赛的时限。
 
-**在官方或其它奖励徽章的竞赛中作弊会使作弊者被永久禁赛。** 这个禁赛不可被申诉（我们不会回复您申请解除禁赛的邮件），且在任何情况下都不会被减期，解除或赦免。
+**在官方或其它奖励徽章的竞赛中作弊会使作弊者被永久禁赛。**这个禁赛不可被申诉（我们不会回复您申请解除禁赛的邮件），且在任何情况下都不会被减期，解除或赦免。
 
 一些玩家会在解除限制并游玩以后立刻被再度限制。这是由机器人完成的，且一般会在 24 小时以内被解除。如果您遇到了这种情况，请迅速通过在推特上通知 [@osusupport](https://twitter.com/osusupport) 或发邮件的方式说明问题。
```
Comment (cl8n) [inline wiki/Help_centre/Account_restrictions/zh.md:77]: in EN it also mentions email as an option in the last sentence

**Review by cl8n — CHANGES_REQUESTED** [review]

**Review by cl8n — APPROVED** [review]

**Pull request merged**